Source/DafnyDriver/CSVTestLogger.cs

```diff
@@ -70,10 +70,11 @@ public class CSVTestLogger : ITestLoggerWithParameters {
     }
 
     private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e) {
-      writer.WriteLine("TestResult.DisplayName,TestResult.Outcome,TestResult.Duration,TestResult.ResourceCount");
+      writer.WriteLine("TestResult.DisplayName,TestResult.Outcome,TestResult.Duration,TestResult.ResourceCount,RandomSeed");
```

Comment:
I want to double-check that adding a field doesn't break any of the existing infrastructure we have around that processes these CSV files. I'm sure we can make anything forward-compatible, if it isn't already, but I'd like to do that before merging this PR (if it's needed at all).

Reply:
Okay, I'm pretty sure this won't cause any trouble with other tools, since it's an additional field at the end.